client/components/CycleRow/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button } from 'react-bootstrap';
+import { deleteCycle } from '../../actions/cycles';
+
+class CycleRow extends Component {
+
+    deleteCycle() {
+        const { id } = this.props.cycle;
+
+        // TODO: How do we call the delete cycle action..? :/
```

Comment:
@evmiguel, tbh I'm not sure whats the best way to dispatch the `deleteCycle` action... how do I get the `dispatch` function at this level?:/